app/layout.tsx

```
@@ -104,8 +109,36 @@ export default async function RootLayout({
+              applicationCategory: "LifestyleApplication",
+              operatingSystem: "Any",
+              offers: {
+                "@type": "Offer",
+                price: "0",
+                priceCurrency: "KRW",
+              },
+              creator: {
+                "@type": "Organization",
+                name: "Oneiri Team",
+              },
+              potentialAction: {
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)